app/services/preserved_object_handler_results.rb

```diff
@@ -85,15 +95,34 @@ def result_hash(code, msg_args=nil)
     { code => result_code_msg(code, msg_args) }
   end
 
-  # result_array = [result1, result2]
-  # result1 = {response_code => msg}
-  # result2 = {response_code => msg}
-  def log_results
+  # output results to Rails.logger and send errors to WorkflowErrorsReporter
+  # @return result_array
+  def report_results
+    candidate_workflow_results = []
     result_array.each do |r|
       severity = self.class.logger_severity_level(r.keys.first)
       msg = r.values.first
       Rails.logger.log(severity, msg)
```

Comment:
I wonder if the code would be clearer if the above 3 lines were in a separate method `log_result` (singular)